docs/40_node-operation/10_getting-started/40_peering.md

```diff
@@ -1,4 +1,165 @@
+
+## What is Peering?
+
+Peering allows EOS nodes to propagate and synchronize the distributed blockchain state by receiving and relaying blocks and/or transactions to other nodes. Any node that is configured to send and receive data in a peer-to-peer fashion is considered a "peer". This adds redundancy and allows for faster response times to client queries and node requests. Therefore, peering is key to the decentralized operation and incremental growth of the EOS blockchain.
+
+> ℹ️ Peering is enabled through the EOS peer-to-peer (p2p) network protocol and it is what makes possible the decentralized operation of the blockchain.
+
+## Why you need nodes/peers?
+
+In the EOS blockchain, nodes can be configured to operate in different ways. It is this flexibility in having nodes serve distinct roles which allows for a more distributed load and a smoother blockchain experience to the end user. Some of the node types in EOS include, but are not limited to:
+
+* **Producing Nodes**: produce blocks to be added to the chain
+* **Relay Nodes**: validate/relay blocks and/or transactions
+* **API Nodes**: respond to API queries from clients via HTTP
+* **History Nodes**: stores chain data for L2 history solutions
+* etc.
+
+Therefore, when set up as peers, EOS nodes validate the blocks and transactions they receive and relay them to other peers by default, if valid. Nodes can also be set up to respond to API requests from clients, provide historical data about blocks and transactions, etc. This separation of concerns makes the blockchain more efficient.
+
+## How to set up peers
+
+> ℹ️ The peer connection process must be performed on the local environment of each peer. Therefore, peering involves planning and agreement among some of the nodes that will act as peers in the EOS network.
+
+Peering can be set up by configuring the `net_plugin` of each `nodeos` instance whose node will act as a peer. The most important options are:
+
+* `p2p-listen-endpoint arg`: local `host:port` for incoming p2p connections
+* `p2p-server-address arg`: public `host:port` for incoming p2p connections
+* `p2p-peer-address arg`: local or remote peer `host:port` to connect to
+
+Other options are available to limit the maximum number of connections, whitelisting specific peers by public key, accept/relay transactions, etc. Check the `net_plugin` options in `nodeos` for more information.
+
+### Peer setup using `config.ini`
+
+To peer your local node with other nodes, specify the following in your nodeos `config.ini` *before* launching your `nodeos` instance:
+
+```ini
+# your listening host:port
```

Comment:
A description of the differences between the listening and public would be good

Reply:
Added